fmriprep/workflows/bold/registration.py

```diff
@@ -32,6 +33,57 @@
 
 DEFAULT_MEMORY_MIN_GB = 0.01
 
+class PatchedBBRegisterRPT(BBRegisterRPT):
+    def _list_outputs(self):
+        outputs = super(PatchedBBRegisterRPT, self)._list_outputs()
+
+        if not op.exists(outputs['out_lta_file']):
+            return outputs
+    
+        with open(outputs['out_lta_file'], 'r') as f:
+            lines = f.readlines()
+    
+        fixed = False
+        newfile = []
+
+        for line in lines:
+            if line.startswith('filename = ') and len(line.strip("\n")) >= 255:
+                fixed = True
+                newfile.append('filename = path_too_long\n')
+            else:
+                newfile.append(line)
+    
+        if fixed:
+            with open(outputs['out_lta_file'], 'w') as f:
+                f.write(''.join(newfile))
+        return outputs
+    
+class PatchedMRICoregRPT(MRICoregRPT):
+
+    def _list_outputs(self):
+        outputs = super(PatchedMRICoregRPT, self)._list_outputs()
+
+        if not op.exists(outputs['out_lta_file']):
+            return outputs
+    
+        with open(outputs['out_lta_file'], 'r') as f:
+            lines = f.readlines()
+    
+        fixed = False
+        newfile = []
+
+        for line in lines:
+            if line.startswith('filename = ') and len(line.strip("\n")) >= 255:
+                fixed = True
+                newfile.append('filename = path_too_long\n')
+            else:
+                newfile.append(line)
+    
+        if fixed:
+            with open(outputs['out_lta_file'], 'w') as f:
+                f.write(''.join(newfile))
+        return outputs
```

Comment:
We keep interfaces and workflows separate.

These should go in [fmriprep/interfaces/freesurfer.py](https://github.com/poldracklab/fmriprep/blob/master/fmriprep/interfaces/freesurfer.py).